### PR TITLE
Image Gallery Plugin Initial UI

### PR DESCRIPTION
**Pull request by shn-srl**
Linear Issue: [Link](https://linear.app/serlo/issue/PE-51/initial-plugin-ui)

**[Demo Link](https://frontend-git-pe-51-initial-plugin-ui-image-gallery-serlo.vercel.app/___editor_preview)**

### CODE REVIEW

**Review by hejtful — APPROVED** [review]
Looks good to me

Maybe just check with Gregor about plugin focused state, should the plugin be shifted to the right a bit to show the vertical line fully? (it's currently only sticking out a bit on the corners)
Screenshot 2024-08-21 at 18 40 07

File: packages/editor/src/plugins/image-gallery/editor.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import type { ImageGalleryProps } from '.'
+import { AddImages } from './components/add-images'
+import { ImageGalleryToolbar } from './toolbar'
+
+export function ImageGalleryEditor(props: ImageGalleryProps) {
+  const { focused } = props
+
+  return (
+    <div
+      className="group/image-gallery mx-auto"
```
Comment (hejtful) [inline packages/editor/src/plugins/image-gallery/editor.tsx]: This group might not be necessary?

Comment (shn-srl) [reply packages/editor/src/plugins/image-gallery/editor.tsx]: Thanks, removed

**Review by elbotho — APPROVED** [review]
👍

**Comment by shn-srl** [comment]
> Looks good to me
> 
> Maybe just check with Gregor about plugin focused state, should the plugin be shifted to the right a bit to show the vertical line fully? (it's currently only sticking out a bit on the corners) Screenshot 2024-08-21 at 18 40 07

Created an issue, this is an existing problem with the image plugin as well:

https://linear.app/serlo/issue/PE-79/fix-grey-focus-sidebar-visibility-in-image-imagegallery-plugins

File: packages/editor/src/editor-ui/assets/plugin-icons/image-gallery/icon-layered-images.svg
Comment (elbotho) [inline packages/editor/src/editor-ui/assets/plugin-icons/image-gallery/icon-layered-images.svg:1]: @shn-srl I missed that yesterday. I don't like svgs with vectorised text. That's bad for file size, reusability, i18n and a11y. Could you change the svg to only include the image stack and add the text and card with in react?

**Pull request merged**